### DISCUSSION TITLE
i got an error

### DISCUSSION THREAD

**Discussion by D0liny** [SKCraft/Launcher#511] [I need help!]
i got this when i try to build, i have th jdk 8 installed as JAVA_HOME
![image](https://github.com/SKCraft/Launcher/assets/139342746/b3e0f7ef-143f-4e8c-ad9b-03de97e75d89)

**Comment by hedgehog1029** [comment]
Transient issue caused by my server rebooting and my Maven going down. Should be fixed now.
